src/app/wallet/wallet.component.ts

```diff
@@ -51,12 +53,14 @@ export class WalletComponent implements OnInit {
             SpinnerUtil.hideSpinner();
             displayBackendError(err);
         });
+        this.topbarService.walletInInitProcess(true);
```

Comment:
Don't do it this way. Go to `TopbarReminderComponent` and subscribe to route changes. When you land on wallet initialization or user identification screen, hide the reminder.